tst/unit/test_container_iterator.cpp

```diff
@@ -290,6 +290,13 @@ TEST_CASE("Can pull variables from containers based on Metadata", "[ContainerIte
         REQUIRE(!intervals_intersect(imap["v3"], imap["vsparse"]));
       }
     }
+
+    WHEN("we add a 2d variable") {
+      std::vector<int> twod_block_size{16, 16, 1};
```

Comment:
Is there any reason we would pass in something like {16, 1, 16} ?

Reply:
I don't think so.  To be honest, most development done on top of parthenon won't ever have to even specify the grid dimensions.  The intent is that developers specify an association like cell, face, node, ... and parthenon takes care of the rest.  That said, I'm actually not sure what would happen if you passed in {16,1,16}.  It might (probably) just says OK this is 3D and everything would work just fine.

Reply:
I don't think `{16, 1, 16}` should ever happen. That would mean `dim2=1` and `dim3=16`. I think the code assumes the trailing index is always the absent one.

Reply:
I'm just wondering if the user will get useful feedback if they try to do something like that, if not it might be worth addressing in the future. This pull request addresses the relevant issue, so I'm happy.

Reply:
Hmm. If it comes up, we can add a warning in `Container.Add` where this shows up.